src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -33,6 +33,7 @@
         <parameter key="translation.loader.class">Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader</parameter>
         <parameter key="translation.extractor.class">Symfony\Component\Translation\Extractor\ChainExtractor</parameter>
         <parameter key="translation.writer.class">Symfony\Component\Translation\Writer\TranslationWriter</parameter>
+        <parameter key="translation.warmer.class">Symfony\Bundle\FrameworkBundle\CacheWarmer\TranslationsCacheWarmer</parameter>
```

Comment:
I'm not sure to understand. You ask me to put my class name directly in my service declaration, that's right ?
If that's right, can you please explain me (for my personnal knowledge) why should I do it in this particular case ?

Thx,